### DISCUSSION TITLE
Window not auto resizing to fit contents

### DISCUSSION THREAD

**Discussion by itslyn** [AvaloniaUI/Avalonia#15584] [Q&A]
I have a window that is not auto resizing to fit its contents. Instead, it is filling up my entire screen (both monitors) so its presumably some sort of max window size.
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/72238cb7-0a68-4e2d-a9f7-6fec20bec8d6)

Here is my XAML. I've tried setting the SizeToContent property on the Window but it seems to not work.
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/b191a4f7-796f-47cc-b23f-a556d9e1bde8)

**Comment by maxkatz6** [comment]
Without more info, I think it's either a problem with RelativePanel or a linux distro support (if it's a linux).
